src/store/actions.js

```diff
@@ -13,9 +13,10 @@ import {
 import { i18n } from './plugins/languages';
 
 export default {
-  switchNetwork({ commit }, payload) {
+  async switchNetwork({ commit, dispatch }, payload) {
     commit('switchNetwork', payload);
     commit('setTransactions', []);
+    await dispatch('fungibleTokens/getAvailableTokens');
```

Comment:
I'm proposing to make `fungibleTokens` a plugin and to subscribe for network updates from there instead of making network switcher aware of multiple parts of the app.

Reply:
Actually that's done in #1016 so I'm closing this one.